contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
     "API Keys": "Yes",
     "Project Support": "Yes",
     "I18N Localized Content": "Yes",
     "Content Trees": "No",
     "Tagging": "Yes",
-    "Content Relations": "Yes",
-    "Nesting of Fields": "No",
-    "GDPR Compliance": "No",
-    "Editing Conflict Detection": "No",
+    "Content Relations": "1:1, 1:n, n:n, n:1",
+    "Nesting of Fields": "Yes",
+    "GDPR Compliance": "Yes",
```

Comment:
https://www.contentful.com/blog/2018/05/24/contentful-gdpr/

Reply:
> Is the system GDPR compliant? Is it possible to quickly find and anonymize stored contents which contain user specific information (e.g. Name, Address, Date of Birth). Summarize compliance support.

Does contentful provide means to achieve this?

Reply:
Regarding personal data for which Contentful is a controller, Contentful does have mechanisms and processes for removing user specific data upon request as required by GDPR. This refers to the name, email address, and any other personal data that Contentful requires in order to use its services. Essentially if a user wants to remove her/his personal data from Contentful, s/he has the ability to delete her/his user account which deletes all such personal data from Contentful. 

Regarding personal data for which Contentful is a processor, Contentful offers data processing agreements to its customers upon request and will act upon the controller's authorized instructions to remove such data as legally required. This includes all content that is uploaded and stored in Contentful; Contentful is the processor of such data and the user which uploaded and stored such data in Contentful is the controller. Essentially, Contentful does not control the data that is stored in the platform and thus the user who stored the data in Contentful is primarily responsible as the controller of that data for ensuring compliance with GDPR.

Reply:
> Regarding personal data for which Contentful is a processor…
> … thus the user who stored the data in Contentful is primarily responsible …

The main idea behind the field in the sheet is to check whether the CMS provides any help in dealing with GDPR (personal) related data. 

I think it would be best to rename the field or add another field which indicates whether the CMS provides help dealing with GDPR (personal data). As a admin I would like to know whether I can quickly look-up and delete or partially anonymize personal data. Kentico Cloud for example provides a personal data API to help dealing with that data.

I'll update the sheet once I found a better name. Any suggestions would be welcome.

Reply:
Makes sense and I can understand where you're coming from. My suggestion would be to split this field into 2 separate fields:

1. Legal GDPR Compliance - like what Contentful (and many others) have done, this represents that the company can operate legally in compliance with applicable GDPR regulations. There is a Data Protection Officer selected and the company has done the legal due diligence to withstand scrutiny with regard to GDPR.
2. Technical features to assist with customer GDPR compliance - these are features which help users of the service/platform be compliant with GDPR regulations. Kentico Cloud's Personal Data API would be an example of this. 

Hope this helps!

Reply:
I have split the existing field up into two parts and I updated the contentful file.